src/Pos/Explorer/Web/Doc.hs

```diff
@@ -1,57 +1,83 @@
+                                                 DocQueryParam (..), Endpoint,
+                                                 ExtraInfo (..),
+                                                 ParamKind (Normal),
+                                                 ToCapture (toCapture),
+                                                 ToParam (toParam),
+                                                 ToSample (toSamples),
+                                                 apiEndpoints, apiIntros,
+                                                 capDesc, capSymbol, captures,
+                                                 defAction, defEndpoint,
+                                                 defaultDocOptions, docsWith,
+                                                 introBody, introTitle,
+                                                 markdown, method, noteBody,
+                                                 notes, paramDesc, paramName,
+                                                 params, path, pretty)
+import           Universum
 
 
 walletDocs :: API
-walletDocs = docsWithIntros intros (Servant.Docs.pretty explorerApi)
--- walletDocs = docsWith defaultDocOptions intros extras (pretty explorerApi)
+walletDocs = docsWith defaultDocOptions intros extras (Servant.Docs.pretty explorerApi)
+-- walletDocs = docsWithIntros intros (Servant.Docs.pretty explorerApi)
 -- walletDocs = docs (pretty explorerApi)
 
 walletDocsText :: Text
 walletDocsText = toText $ markdown walletDocs
 
+walletTableDocsText :: Text
+walletTableDocsText = toText $ markdownTable walletDocs
+
 intros :: [DocIntro]
-intros =
-    [ DocIntro
-          "Documentation of cardano-explorer web API"
-          ["This is very first version, don't expect it to be smart."]
+intros = [DocIntro "Explorer Backend API"
+    [ "Currently, the explorer's API provides a series of methods to work with `cardano-sl`. The `servant` Haskell library that provides a modular approach to API-building was used. This library uses combinators to both build atomic HTTP actions and to glue these atomic methods together to form larger and more complete APIs."
+    , "If the event requests fail, there is a `ExplorerError` type, which is simply a wrapper over `Text` to show what happened."
+    , "Currently, the explorer's API supports the following operations (see Comments below):"]]
+
+extras :: ExtraInfo api
+extras =
+    ExtraInfo . HM.fromList $
+    [ (defEndpoint  & path <>~ ["api", "blocks", "last"], defAction & notes <>~ [ DocNote "Description" ["Get last block."] ])
+    , (defEndpoint  & path <>~ ["api", "blocks", "summary", ":hash"], defAction & notes <>~ [ DocNote "Description" ["Get block summary."] ])
+    , (defEndpoint  & path <>~ ["api", "blocks", "txs", ":hash"], defAction & notes <>~ [ DocNote "Description" ["Get block transactions."] ])
+    , (defEndpoint  & path <>~ ["api", "txs", "last"], defAction & notes <>~ [ DocNote "Description" ["Get last transaction."] ])
```

Comment:
Although the generated `cardano-explorer-table-web-api.md` describes `Optional parameters` in a column, I would recommend to add all optional params to the endpoint, too. 

For example: `["api", "txs", "last", ":limit", ":offset"]` (generated `/api/blocks/last/:limit/:offset`) instead of `["api", "txs", "last"]` (generated `/api/blocks/last/`) to reflect all parameters of endpoints in the same way as `Api.hs` it does. 
``